glslc/README.asciidoc

```diff
@@ -23,9 +23,9 @@ glslc [-c|-S|-E]
 
 == Description
 
-=== Shader stage specification
+=== Input dialect specification
```

Comment:
As discussed offline, it's confusing to name the section "Input dialect" and then dive straight into describing shader-stage selection.